include/ck/tensor_operation/gpu/device/device_elementwise_extension.hpp

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <memory>
+#include <array>
+
+#include "ck/ck.hpp"
+#include "ck/utility/data_type.hpp"
+#include "ck/utility/tuple.hpp"
+#include "ck/utility/math_v2.hpp"
+
+#include "ck/tensor_operation/gpu/device/device_elementwise.hpp"
+
+namespace ck {
+namespace tensor_operation {
+namespace device {
+
+struct NormalizeInInfer
```

Comment:
I think this should be implemented as elementwise operation

Reply:
Do you mean move this to the file `include/ck/tensor_operation/gpu/element/element_wise_operation.hpp` ?

Reply:
Yes, I mean moving it into ```element_wise_operation.hpp```

Reply:
Solved by commit [67203](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/531/commits/9b04b3b920de0195a522a5cef735287283a67203)